src/plugin.ts

```diff
@@ -0,0 +1,33 @@
+import { Web3PluginBase, utils } from "web3";
+import { AddEthereumChainRequest } from "./types";
+
+type WalletRpcApi = {
```

Comment:
Why isn't this defined in `types.ts`?

Reply:
all types from `types.ts` are exported from the plugin package to use outside of it, this one is for internal use only.